scripts/benchmarks/csv-json-converter.ts

```diff
@@ -63,7 +63,9 @@ function convertCsvToJson(csvPath: string, performanceYear: number, jsonFileName
         
         //populate some default values if they are not found in the csv.
         measure.isToppedOutByProgram = measure.isToppedOutByProgram || false;
-        measure.averagePerformanceRate = measure.averagePerformanceRate || null;
```

Comment:
This code change is to auto update the averagePerformanceRate value and set it as a number instead of a string.